skopt/learning/__init__.py

```diff
@@ -4,6 +4,17 @@
 from .forest import ExtraTreesRegressor
 from .gbrt import GradientBoostingQuantileRegressor
 from .gpr import GaussianProcessRegressor
+from .gp_kernels import ConstantKernel
+from .gp_kernels import DotProduct
+from .gp_kernels import Exponentiation
+from .gp_kernels import ExpSineSquared
+from .gp_kernels import Matern
+from .gp_kernels import Product
+from .gp_kernels import RationalQuadratic
+from .gp_kernels import RBF
+from .gp_kernels import Sum
+from .gp_kernels import WhiteKernel
```

Comment:
Yes! `__all__` is just used for the `*` wildcard while importing. I'll change it anyway